src/quequations.jl

```diff
@@ -0,0 +1,54 @@
+abstract QuEquation
+
+immutable SchrodingerEquation{H<:QuBase.AbstractQuMatrix} <: QuEquation
+    hamiltonian::H
+    SchrodingerEquation(hamiltonian) = new(hamiltonian)
+end
+
+SchrodingerEquation{H<:QuBase.AbstractQuMatrix}(hamiltonian::H) = SchrodingerEquation{H}(hamiltonian)
+
+immutable LiouvillevonNeumannEquation{H<:QuBase.AbstractQuMatrix} <: QuEquation
+    liouvillian::H
+    LiouvillevonNeumannEquation(superoperator) = new(superoperator)
+end
+
+LiouvillevonNeumannEquation{H<:QuBase.AbstractQuMatrix}(liouvillian::H) = LiouvillevonNeumannEquation{H}(liouvillian)
+
+function liouvillian_op(h::QuBase.AbstractQuMatrix)
+    nb = size(coeffs(h), 1)
+    SI = Array(Int,0)
+    SJ = Array(Int,0)
+    Lvals = Array(typeof(im*one(eltype(h))),0)
+    for m=1:nb
+        for n=1:nb
+            for i=1:nb
+                for j=1:nb
+                    sm = (n-1)*nb + m
+                    sj = (j-1)*nb + i
+                    lv = zero(Complex128)
+                    if j==n
+                        lv = lv - im*h[m,i]
+                    end
+                    if i==m
+                        lv = lv + im*h[j,n]
+                    end
+                    if real(lv)!=0 || imag(lv)!=0
+                        push!(SI, sm)
+                        push!(SJ, sj)
+                        push!(Lvals, lv)
+                    end
+                end
+            end
+        end
+    end
+    return QuBase.similar_type(h)(sparse(SI, SJ, Lvals, nb*nb, nb*nb))
+end
+
+function liouvillian_tensor(h::QuBase.AbstractQuMatrix)
+    return -im*(tensor(eye(h), h) - tensor(h',eye(h)))
+end
+
+export SchrodingerEquation,
+      LiouvillevonNeumannEquation,
+      liouvillian_op,
```

Comment:
Where are they used in `propmachinery.jl`? But even if they are, there is no need to export them since this file and `propmachinery.jl` are in the same module.